src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -10,6 +10,8 @@
 #include "CascadiaSettings.h"
 #include "Profile.h"
 
+#include <optional>
```

Comment:
nit: I think this _should_ already be in our `pch.h`, by way of `LibraryIncludes.h`, so this _shouldn't_ be necessary

Reply:
Fixed :-)  Thx!